### ISSUE TITLE
Stop download once an existing book is encountered

### ISSUE THREAD

**Issue by fref** [BurnhamG/packtpub-downloader#12]
I'm downloading the books only once in a while, and then I either delete some of them, or put them on my ereader, or store them elsewhere.

I'd be nice to have a flag or something to say to the downloader: stop reading the books once you reach a book in the list that is already downloaded (that way, I'd only need to keep the most recent book in the download folder after I moved the books around).

**Comment by BurnhamG** [comment]
This is a good idea - I'll definitely take a look at adding this feature.
